src/libs/actions/App.js

```diff
@@ -35,15 +33,15 @@ Onyx.connect({
 
 let myPersonalDetails;
 Onyx.connect({
-    key: ONYXKEYS.PERSONAL_DETAILS,
+    key: ONYXKEYS.PERSONAL_DETAILS_LIST,
     callback: (val) => {
-        if (!val || !currentUserEmail) {
+        if (!val || !currentUserAccountID) {
             return;
         }
 
-        myPersonalDetails = val[currentUserEmail];
-    },
-});
+        myPersonalDetails = val[currentUserAccountID];
+    }
+})
```

Comment:
This is probably what you're missing 😅 

```suggestion
    },
});
```

Reply:
Yes, that was the case - I was certain the prettier task is called before every push... Fixed now!